src/utils/AddressUtils.ts

```diff
@@ -0,0 +1,18 @@
+import { providers } from "ethers";
+
+/**
+ * Checks if a contract is deployed at the given address
+ * @param address The ETH address to check
+ * @param provider A valid Ethers.js provider
+ * @returns A boolean indicating if a contract is deployed at the given address or not (true = contract, false = no contract)
+ */
+export async function isContractAddress(address: string, provider: providers.Provider): Promise<boolean> {
+  // A base case for if the address is null
+  if (!address) {
```

Comment:
This just makes it easier for me to not have to add a conditional when trying to call this function in the event that the calling fn's address is potentially undefined